src/VirtoCommerce.CatalogPublishingModule.Core/Model/CompletenessChannel.cs

```diff
@@ -20,6 +21,10 @@ public class CompletenessChannel : AuditableEntity
 
         public string EvaluatorType { get; set; }
 
-        public int CompletenessPercent { get; set; }
+        public decimal? CompletenessPercent { get; set; }
+        public object Clone()
```

Comment:
Separate property and method with an empty line.
This also applies to CompletenessDetail and CompletenessEntry.